### ISSUE TITLE
Installation instructions for JabRef 4.x not up to date for Ubuntu 20.04

### ISSUE THREAD

**Issue by teertinker** [JabRef/jabref#6341]
Recently Ubuntu 20.04 was released.
I upgraded my system and tried to install JabRef. Installation of JabRef 5 works without any problems. However, due to its high memory footprint and missing features I'd like to install JabRef 4.3 as an independent jar file.

I followed the instructions at: https://docs.jabref.org/general/installation#jabref-4-x
but they seem to be outdated. 
1. I didn't manage to install Oracle Java 8 since the refered ppa is not maintained any more (ppa:webupd8team/java)
2. I did manage to install Open JDK for Java 8, but installing openjfx did not work the way described.
"sudo apt install openjfx=8u161-b12-1ubuntu2 libopenjfx-jni=8u161-b12-1ubuntu2 libopenjfx-java=8u161-b12-1ubuntu2" 
there is not such package.

If there is a solution, I would like to update the docs.

**Comment by AEgit** [comment]
If the performance/memory footprint is the main issue: Would the current dev version be a solution for you? There have been some major performance improvements lately:
https://github.com/JabRef/jabref/issues/5919

You can easily install the snap package on Ubuntu (https://snapcraft.io/jabref) and switch to the latest/edge channel. That gives you the most up to date version.
Always make sure to have a backup of your database, since you are using a bleeding edge version.

Having said, if there are obviously features that are missing from JabRef at the moment or if you are not happy with its stability at the moment, then this might not be a solution to you.

**Comment by teertinker** [comment]
Thank you very much! I will give the dev-snapshot a try.

Meanwhile I managed to get 4.3.1 working by using Oracle JRE.
1. I had to download the "Linux x64" tar.gz archive from: https://www.java.com/de/download/linux_manual.jsp
2. Unpack the archive e.g. to the Downloads folder
3. Open Terminal 
4. Go to the folder, JabRef-x.jar is located
Start the Jabref-X.jar file via terminal by directly referring to the java JRE :
`/home/USER/Downloads/jre-8u251-linux-x64/jre1.8.0_251/bin/java -jar JabRef-4.3.1.jar `

- This rather complicated way seems to be necessary, because I didn't manage to get Ubuntu to recognize the JRE systemwide. 

May be there is a more user friendly way - that can be used to update the docs.
